Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -396,4 +396,41 @@
     <!-- XHED: Cov Pass Info Fragment title -->
     <string name="cov_pass_info_section_three">"Bitte beachten Sie, dass in anderen Ländern gegebenenfalls andere Apps zur Zertifikatsprüfung durch Dritte verwendet werden."</string>
 
+    <!-- #####################################################
+        CONFIRMED STATUS CARD
+        ##################################################### -->
+
+    <!-- XTXT: Confirmed status card title -->
+    <string name="confirmed_status_title">"Status-Nachweis"</string>
+
+    <!-- XTXT: Confirmed status card 2g+ PCR subtitle -->
+    <string name="confirmed_status_2g_pcr_subtitle">"2G+ PCR-Test"</string>
+    <!-- XTXT: Confirmed status card 2g+ RAT subtitle -->
+    <string name="confirmed_status_2g_rat_subtitle">"2G+ Schnelltest"</string>
+
+    <!-- XTXT: Confirmed status card 2G badge -->
+    <string name="confirmed_status_2g_badge">"2G"</string>
+    <!-- XTXT: Confirmed status card 2G+ badge -->
+    <string name="confirmed_status_2g_plus_badge">"2G+"</string>
+    <!-- XTXT: Confirmed status card 3G badge -->
+    <string name="confirmed_status_3g_badge">"3G"</string>
+    <!-- XTXT: Confirmed status card 3G+ badge -->
+    <string name="confirmed_status_3g_plus_badge">"3G+"</string>
+
+    <!-- XTXT: Confirmed status card 2G body -->
+    <string name="confirmed_status_2g_body">"Ihre Zertifikate ermöglichen Ihnen Zutritt für 2G Events. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+    <!-- XTXT: Confirmed status card 2G+ body -->
+    <string name="confirmed_status_2g_plus_body">"Ihre Zertifikate ermöglichen Ihnen Zutritt für 2G+ Events. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+    <!-- XTXT: Confirmed status card 3G body -->
+    <string name="confirmed_status_3g_body">"Ihre Zertifikate ermöglichen Ihnen Zutritt für 3G Events. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
+    <!-- XTXT: Confirmed status card 3G+ body -->
+    <string name="confirmed_status_3g_plus_body">"Ihre Zertifikate ermöglichen Ihnen Zutritt für 3G+ Events. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht."</string>
```

Comment:
The text for these 4 strings is the same except for (2G|2G+|3G|3G+), or? What about having only one string and filling the admission state text dynamically with an `%s` placeholder 🤔

Reply:
That was my first thought 😆 But my guts tells me that there would be some text changes...

Reply:
Alright, then let's leave it as is 🙂